src/components/Bookings/Bookings.jsx

```diff
@@ -63,31 +64,71 @@ const Bookings = () => {
       )
     );
   };
+  //error handling for the form inputs
+  const [error, setError] = useState(null);
+
+  // Function to check the email format
+  const isValidEmail = (email) => {
```

Comment:
Why didn't you use the pattern attribute to add validation on email input indead of creating a new funtion?